pyiron_contrib/atomistics/xtb/xtb.py

```diff
@@ -0,0 +1,149 @@
+import os
+from io import StringIO
+import tempfile
+import shutil
+import contextlib
+import pandas as pd
+import numpy as np
+
+from pyiron_base import PythonTemplateJob
+from pyiron import ase_to_pyiron
+
+from ase.optimize import LBFGSLineSearch
+import ase.io
+
+from xtb.ase.calculator import XTB
+
+@contextlib.contextmanager
+def make_temp_directory(prefix=None):
+    """make_temp_directory function to make a temporary directory and change there.
+    Needed for xtb jobs that create input read/written from files.
+
+    Parameters
+    ----------
+    prefix : str, optional
+        path prefix to temporary folder, by default None
+
+    Yields
+    ------
+    temp_dir : str
+        name of the temporary directory
+    """
+    mycwd = os.getcwd()
+    try:
+        temp_dir = tempfile.mkdtemp(prefix=prefix)
+        os.chdir(temp_dir)
+        yield temp_dir
+    finally:
+        os.chdir(mycwd)
+        shutil.rmtree(temp_dir)
+
+def convert_ase_xyz(ase_atoms):
+    """convert_ase_xyz
+
+    Parameters
+    ----------
+    ase_atoms : ase.Atoms
+        ase atoms to write to xyz string
+
+    Returns
+    -------
+    outstring : str
+        xyz file string
+    """
+    outstring = '{}\n\n'.format(len(ase_atoms))
+    for atom in ase_atoms:
+        outstring += '{} {} {} {}\n'.format(atom.symbol,
+                                            atom.position[0],
+                                            atom.position[1],
+                                            atom.position[2])
+    outstring = outstring.strip('\n')
+    return outstring
```

Comment:
Using `StringIO` and `ase_atoms.write()` should work as well, or are you specifically trying to surpress the cell dimensions in the output?